src/shards.rs

```diff
@@ -5,10 +5,13 @@ use crate::error::{JoinError, JoinResult};
 use futures::channel::mpsc::{TrySendError, UnboundedSender as Sender};
 #[cfg(feature = "serenity")]
 use parking_lot::{lock_api::RwLockWriteGuard, Mutex as PMutex, RwLock as PRwLock};
+#[cfg(not(feature = "serenity"))]
 use serde_json::Value;
 #[cfg(feature = "serenity")]
```

Comment:
Could you please combine your new import into this one here?

Reply:
This is possibly a side effect of cargo-fmt